examples/api/e2e/playwright.config.ts

```diff
@@ -1,6 +1,7 @@
 import { defineConfig, devices } from "@playwright/test";
+import nodeProcess from "node:process";
```

Comment:
Doesn't that work:
```suggestion
import type nodeProcess from "node:process";
```

Reply:
no 🤷 

![Screenshot 2025-01-24 at 10 24 22](https://github.com/user-attachments/assets/a5954e63-bd2c-4328-b637-628875905469)

Reply:
(the difference being that `import type` does not import anything (i.e. TS only construct)

Reply:
updated 👍